regions/core/core.py

```diff
@@ -46,8 +46,8 @@ def __xor__(self, other):
         return self.symmetric_difference(other)
 
     def __contains__(self, coord):
-        if not coord.isscalar:
-            raise ValueError('{} must be scalar'.format(coord))
+        # if not coord.isscalar:
+        #     raise ValueError('{} must be scalar'.format(coord))
```

Comment:
Maybe uncomment this check and implement `PixCoord.isscalar`?
See https://github.com/astropy/regions/issues/25#issuecomment-232321254